lib/plug/conn.ex

```diff
@@ -445,10 +446,10 @@ defmodule Plug.Conn do
   end
 
   def send_chunked(%Conn{adapter: {adapter, payload}, owner: owner} = conn, status) do
-    conn = run_before_send(%{conn | status: Plug.Conn.Status.code(status), resp_body: nil}, :chunked)
+    conn = run_before_send(%{conn | status: Plug.Conn.Status.code(status), resp_body: nil}, :set_chunked)
```

Comment:
The Logger pattern matches on `state` to log "Chunked" requests correctly https://github.com/elixir-lang/plug/pull/524/commits/0727147b5494b8d3412e4f9097d676bec8139e07#diff-e05016dae0fb618991697f2223cb8dd0R49

I can keep the state at `:set`, and have the logger match on the transfer encoding header.

Would you prefer this?